packages/devextreme/js/__internal/grids/grid_core/views/m_grid_view.ts

```diff
@@ -921,6 +934,14 @@ export class GridView extends modules.View {
       $groupElement.appendTo($rootElement);
     }
 
+    if (!this._a11yGeneralStatusElement) {
+      this._a11yGeneralStatusElement = A11yStatusContainerComponent(
+        { a11yLiveType: 'polite' },
```

Comment:
According to the documentation. the [status role](https://www.w3.org/TR/wai-aria-1.2/#status) already has implicit aria-live=polite. We can reduce this argument.
![image](https://github.com/DevExpress/DevExtreme/assets/16618553/adbec172-4eda-451d-bdb3-40f23bd9bfc6)

Reply:
Fixed